src/cr/cube/matrix.py

```diff
@@ -128,6 +159,10 @@ def _base_values(self):
         """2D np.float64 ndarray of table-stderr for each cell of cube-result matrix."""
         return getattr(self._cube_result_matrix, self._measure_propname)
 
+    def _subtotal_column(self, subtotal):
+        """Return (n_rows,) ndarray of np.nan values."""
+        return np.sum(self._base_values[:, subtotal.addend_idxs], axis=1)
```

Comment:
You don't really want to be implementing two methods in the same commit. Sometimes you write multiple tests (when the method has two or more very different execution paths, like when it has an exception path), but generally it's one method/property per commit. So you'd be mocking `._subtotal_column` and it's return value(s) in this one.